source/security/IAM/iam-providers.rst

```diff
@@ -0,0 +1,11 @@
+=========
+Providers
+=========
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Stub - might duplicate STS page?
```

Comment:
I guess STS should be moved here. It's part of user authorization. Eventually we need to explain what STS is (purpose, workflow etc.) However, not necessarily in this PR.

Reply:
Hm - yeah lets do that in another PR. We can kind of mention both in kind (External Identity Providers supported via STS) or something).